### PR TITLE
Edits intro to decommissioning a node

### PR DESCRIPTION
**Pull request by emanic**
## Description

- Updates the intro to decommissioning a node as per new node controller
- Also fixes broken link that blocks the merge of this PR and spelling errors (v2.5 and master dirs)

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: master/usage/decommissioning-a-node.md [hunk 1]
```diff
@@ -2,14 +2,21 @@
 title: Decommissioning a Node
 ---
 
-### Why you might be interested in this guide
+### About decommissioning nodes
```
Comment (ozdanborne) [inline master/usage/decommissioning-a-node.md]: Do we want to just nix this header altogether? The title of the page is directly above it and is basically the same thing.

File: master/usage/decommissioning-a-node.md [hunk 2]
```diff
@@ -2,14 +2,21 @@
...
 
-- You are decomissioning a host running calico/node or removing it from your
+If you are running the [the kube-controllers node controller]({{site.baseurl}}/{{page.version}}/reference/kube-controllers/configuration)
```
Comment (ozdanborne) [inline master/usage/decommissioning-a-node.md]: We should make sure we like this term and then  add it to the terminology page.

File: master/usage/decommissioning-a-node.md (hunk 1 above)
Comment (emanic) [reply master/usage/decommissioning-a-node.md]: I know what you mean, but I like to have a heading for every section. It's more orderly, in my opinion...more organized that way.

File: master/usage/decommissioning-a-node.md (hunk 2 above)
Comment (caseydavenport) [reply master/usage/decommissioning-a-node.md]: Potentially "kube-controllers with the node controller enabled"

Comment (caseydavenport) [reply master/usage/decommissioning-a-node.md]: Or just "running the node controller"

Comment (emanic) [reply master/usage/decommissioning-a-node.md]: It is a good point @ozdanborne ....I think `kube-controller` is the name of the image and we have actually standardized on "node controller" as the plain English equivalent, so I like Casey's second suggestion.

**Pull request merged**